resources.md

```diff
@@ -37,7 +36,14 @@ title: Resources
 - [Ontofox](http://ontofox.hegroup.org/): An ontology term and relation extraction and reuse tool.
 - [Ubergraph](https://github.com/INCATools/ubergraph): A sparql endpoint with many OBO ontologies loaded and pre-reasoned with simple triples materialized
 
+## Ontology Analysis
+
+- [OBO Dashboard](https://dashboard.obofoundry.org): An assesment of OBO Foundry ontologies' conformance to OBO Foundry principles 
+- [OBO Community Health Report](https://github.com/cthoyt/obo-community-health): A self-updating assessment of the quality of metadata, responsiveness of the maintainers, and the overall community engagement for each OBO Foundry ontology.
+- [Ontology Quality Assessment](https://github.com/cthoyt/oquat): A self-updating assesment of the semantic quality of OBO Foundry ontologies and beyond (i.e., using known prefixes, using standard identifiers, etc.) 
```

Comment:
I would prefer to link the service rather than the source code for both projects

https://cthoyt.com/oquat/

Reply:
Done